translib/transformer/test/testxfmryang_test.go

```diff
@@ -0,0 +1,355 @@
+//////////////////////////////////////////////////////////////////////////
```

Comment:
Better to keep it in the transformer directory itself

Reply:
Addressed as suggested